BackEnd/src/main/java/com/javaschool/service/BackEndServiceImp.java

```diff
@@ -0,0 +1,29 @@
+package com.javaschool.service;
+
+import com.javaschool.entitymapper.PackageType;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class BackEndServiceImp implements BackEndService {
+
+    private QueueResponseService queueResponseService;
+
+    public BackEndServiceImp(final QueueResponseService queueResponseService) {
+        this.queueResponseService=queueResponseService;
+    }
+
+    @Override
+    public List<String> getType() {
+        List<PackageType> typesResponse = queueResponseService.getType();
+
+        List<String> listOfTypes = new ArrayList<>();
+        for (PackageType type : typesResponse) {
```

Comment:
Check the stream API to process collections. You can replace most of the logic in this method by:

```
return shippingService.getPackageTypes()
                 .stream()
		.map(packageType->packageType.getDescription())
		.collect(Collectors.toList());
```